src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -8,7 +8,9 @@ import {atom, useRecoilState} from 'recoil';
 import AnonymousMessageRepo from '../../../service/repo/anonymousMessageRepo';
 import ChannelList from '../../../database/schema/ChannelListSchema';
 import SignedMessageRepo from '../../../service/repo/signedMessageRepo';
-import UseChatUtilsHook from '../../../../types/hooks/screens/useChatUtilsHook.types';
+import UseChatUtilsHook, {
+  ContactScreenPayload
+} from '../../../../types/hooks/screens/useChatUtilsHook.types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been modified to include the `ContactScreenPayload` type. This is a good practice as it ensures that the payload passed to the `goToContactScreen` function is of the correct type, improving code reliability and maintainability.
```diff
- import UseChatUtilsHook from '../../../../types/hooks/screens/useChatUtilsHook.types';
+ import UseChatUtilsHook, {
+   ContactScreenPayload
+ } from '../../../../types/hooks/screens/useChatUtilsHook.types';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->